app/src/main/java/org/mifos/selfserviceapp/ui/activities/HomeActivity.java

```diff
@@ -42,6 +49,12 @@
     @Inject
     PreferencesHelper preferencesHelper;
 
+    @Inject
+    UserDetailsPresenter detailsPresenter;
+
+    TextView username;
```

Comment:
Good point, will update the naming of views and yeah the views can be made private